build/shade/_k-standard-goals.shade

```diff
@@ -25,9 +25,9 @@ default SAMPLES_PROJECT_GLOB = "samples/*/project.json"
   {
     E("DOTNET_BUILD_VERSION", BuildNumber);
   }
-  if (string.IsNullOrEmpty(E("DOTNET_AUTHOR")))
+  if (string.IsNullOrEmpty(E("NUGET_AUTHOR")))
   {
-    E("DOTNET_AUTHOR", AUTHORS);
+    E("NUGET_AUTHOR", AUTHORS);
```

Comment:
Could we just hard code the value for this?  `E("NUGET_AUTHORS", "Microsoft Corporation");`? It would allow us to remove that line of code from individual makefiles.